### PR TITLE
Serialize status with include list

### PR DESCRIPTION
**Pull request by jfhuete**
Doofinder part of issue: https://github.com/peillis/js-layers/issues/652

Serilize status with include list and removed page = 1 as defautl when page is loading.

### CODE REVIEW

File: src/controller.coffee [hunk 1]
```diff
@@ -60,7 +60,7 @@ class Controller extends EventEnabled
   ###
   reset: (query = null, params = {}, items = []) ->
     @query = query
-    @params = merge {}, @defaults, params, page: 1
+    @params = merge {}, @defaults, params
```
Comment (carlosescri) [inline src/controller.coffee]: If you want to be able to reset page too, just put it in the first parameter of `merge`, like:

```coffeescript
@params = merge page: 1, @defaults, params
```

Comment (jfhuete) [reply src/controller.coffee]: But the change is that "page: 1" is removed. Page not have to be reset.

Comment (carlosescri) [reply src/controller.coffee]: - `page` MUST be `1` if you don't specify another value.
- `reset()` was forcing `page` to always be `1` so the change must ensure the previous requirement is complied, but allow another value to be set if desired. That's why your code is wrong for me.

The `serializeStatus()` method was removing `page` so you're now allowing passing a white list of parameters that shouldn't be removed. OK, then, when de-serializing status you will pass your `page` parameter to the `reset()` method and it should accept it.

But you can't break the default behavior or all layers will stop working properly.

Comment (jfhuete) [reply src/controller.coffee]: done

**Pull request merged**